### ISSUE TITLE
psr/event-dispatcher is incompatible with symfony 4

### ISSUE THREAD

**Issue by pinoniq** [web-token/jwt-framework#181]
It is currently impossible to use the bundle provided in v2 with symfony. However this one is installed by default.

This because of a dependency on `Psr\EventDispatcher\EventDispatcherInterface`

The error is:
```
 Cannot autowire service "Jose\Bundle\JoseFramework\Services\HeaderCheckerManagerFactory": argument "$eventDispatcher" of method "__construct()" references interface "Psr\EventDispatcher\EventDispatcherInterface" but no such service exists. You should   
  maybe alias this interface to one of these existing services: "debug.event_dispatcher", "debug.event_dispatcher.inner".             
```

**Comment by Spomky** [comment]
It is compatible, but as spotted out an alias have to be created:
Be sure that
* `psr/event-dispatcher` is installed (should be with v2.0.5+)
* `symfony/event-dispatcher` (you may require it depending on your SF app).

Second, create the following alias in `services.yaml`:

```yaml
services:
    Psr\EventDispatcher\EventDispatcherInterface:
        alias: 'event_dispatcher'
```

**Comment by Spomky** [comment]
Closing as answered.
Feel free to re-open this issue in needed.

**Comment by eman1986** [comment]
can this be in the documentation? I stumbled on this issue as well and this solved my problem.

**Comment by Spomky** [comment]
Hi @eman1986,

Indeed I will try to automatically create the alias from the bundle if missing.
This way you don't need to modify any config file or read specific notices from the doc.

**Comment by Spomky** [comment]
Fixed in `v2.1.0`
